modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3441,6 +3441,144 @@ Response::
 | String ID of the query.
 |===
 
+
+=== Get information about a template query
+
+`GET /gsql/library?functionName=pkg1.pkg2.<template name>`
```

Comment:
since pkg1 and pkg2 are mutable, should them be surrounded with ‘<’, looks like <pkg1.pkg2.template name>